Godeps/Godeps.json

```diff
@@ -943,6 +943,41 @@
 			"Rev": "db96a2b759cdef4f11a34506a42eb8d1290c598e"
 		},
 		{
+			"ImportPath": "github.com/minio/minio-go",
+			"Comment": "v2.0.4-13-g29b0515",
+			"Rev": "29b05151452ac74ee9b46c93c2200f8b57331e24"
+		},
+		{
+			"ImportPath": "github.com/minio/minio-go/examples/minio",
+			"Comment": "v2.0.4-13-g29b0515",
+			"Rev": "29b05151452ac74ee9b46c93c2200f8b57331e24"
+		},
+		{
+			"ImportPath": "github.com/minio/minio-go/examples/s3",
+			"Comment": "v2.0.4-13-g29b0515",
+			"Rev": "29b05151452ac74ee9b46c93c2200f8b57331e24"
+		},
```

Comment:
These two example packages should not be vendored.

If you are having trouble with vendoring, just `rm -rf Godeps vendor` and then run `godep save ./...`.